tsfc/coffee.py

```diff
@@ -213,7 +213,7 @@ def _expression_division(expr, parameters):
 # Copied from FFCX (ffc/language/ufl_to_cnodes.py)
 math_table = {
     'sqrt': ('sqrt', 'csqrt'),
-    'abs': ('fabs', 'cabs'),
+    'abs': ('fabs', 'abs'),
```

Comment:
COFFEE mode probably not very relevant anymore, but this change does not look right.

Reply:
I cherry-picked from the sprint branch (particularly ff5193e66408d) perhaps @sv2518 can comment on why this change was needed?

Reply:
Oh, I guess it's because the slate-mode kernels were being compiled in C++ mode, for which `abs` is correct. But we decided not to port slate until it is migrated to the loopy codegen path anyway, so this change is moot.

Reply:
Oh yes, complex is quite different in C and in C++...

Reply:
Dropped this part.